### ISSUE TITLE
Minimal hw configuration features?

### ISSUE THREAD

**Issue by AndersBNielsen** [skiselev/monster-fdc#3]
The readme describes the board would work with just a single FDC + address decoding. Looks like an amazingly cheap alternative to buying a vintage FDC card.

Does that mean it would work with any drive the stock BIOS normally supports? (Without the bios extension)

Specifically I’m curious if I can get a 1.44Mb HD drive with an IBM XT 5160 (with 82 or ‘86 BIOS) - maybe just as a DD 720K drive? (Not sure what I can expect from the XT)

Basically, what can I expect from the minimal configuration with one FDC?

**Comment by skiselev** [comment]
- The Monster FDC board will work with the system BIOS. In this case it would only support the first FDC
- System BIOS of IBM PC/XT and compatibles only supports disk up to 720 KB. You can use 3.5"/1.44MB drives, but only with 720 KB diskettes (tape the "density detect" hole, if you like to format a 1.44 MB diskette as 720 KB). These systems typically support up to 4 drives
- System BIOS of IBM AT and compatibles normally only supports 2 drives

If you're looking for a simpler FDC board, you could build or buy a board with only one FDC. For example my [ISA Floppy Disk and Serial Controller](http://www.malinov.com/Home/sergeys-projects/isa-fdc-and-uart), or derivatives, such as [Monotech DeluxeFloppy](https://monotech.fwscart.com/DeluxeFloppy_-_8-bit_ISA_Bootable_HD_Floppy_+_Serial/p6083514_19478745.aspx) or [TexElec Quad-Flop](https://texelec.com/product/quad-flop-four-port-isa-floppy-controller/).

**Comment by AndersBNielsen** [comment]
Thank you so much for getting back to me!  

I have a Monster FDC PCB on my desk and components ready to solder - I might try it both with and without your BIOS. SST39SF010's are cheap! I already used bits and pieces of your Floppy BIOS for my "BIOS extension pong" project.

I see no issue partially populating the PCB with just a single FDC + decoding + passives so that's what I'm going for - that'll certainly be in the price range I'm going for. 
Thanks for the tip with the tape density detect hole - I would've forgotten about that and the 3.5" drives are easier to work with for testing than the 360k's

Thanks again!
